spec/models/purchase_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe Purchase do
+
+  before(:each) do
```

Comment:
Please refactor this into a `let`. The before isn't necessary, and it's also not necessary to query the first Purchase when you can just do:

```ruby
@purchase = Purchase.create({email: "" })
```